### ISSUE TITLE
Create a Device input selection for Camera Input

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#494]
It seemed like much of the problem that @marlitas and I found with using MediaPipe in safari over in https://github.com/phetsims/ratio-and-proportion/issues/472 can be traced back to the fact that PhET wasn't allowing the user to select the available device. I'll try to get that working here.

**Comment by zepumph** [comment]
Ok. Lots of great stuff here!

I was able to get an option into the mediaPipe options in which you can select your media device. I was only able to test on my device which just has a single webcam though. 
As part of this, we had to ditch using camera and control utils (Good riddance!)

It added a bit of complexity to the code, but all of it is in browser API (via mediaDevices) instead of mediaPipe layering that has no unminified code (see https://stackoverflow.com/questions/68963382/where-is-mediapipe-camera-utils-npm-module-source-code) and little documentation about the inner (or outer) workings.

Next step is for @marlitas to test on her computer. When you get to this, please test general loading on chrome and safari, and make sure to also go to the preferences dialog and swap between your devices. I couldn't test this part, so if it is broken I think we should pair to solve it. It shouldn't be too challenging though because it is the exact same code.

Here is a dev version we can use. https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-dev.29/phet/ratio-and-proportion_all_phet.html?camerInput=hands&showVideo

**Comment by terracoda** [comment]
Just an FYI, Camera Input seems to be working on Safari now.

**Comment by terracoda** [comment]
And also, an FYI, if I turn my wireless off after I load "cameraInput=hands" it still seems to work. 
It seems to me that one may only need an internet connection to download the software?

**Comment by terracoda** [comment]
Hmm, my camera is not getting recognized anymore. I am not sure what happened. Maybe device detection is not working perfectly yet. It might be a local setting for my camera, though.

**Comment by marlitas** [comment]
@samreid and I tested this together and saw the following behaviors:
- The camera input would load correctly at startup, and identified an available camera for use.
- When we tried switching camera inputs (both in Chrome and Safari), we got the error: `Error Loading Camera Input: Hands` `Camera Input requires an internet connection to work correctly.` 
- @samreid noticed that the console reported ROI to be 0 when we switched between cameras. `ROI width and height must be > 0.`
- In Chrome it came up with my external usb camera as the default, and in Safari it came up with my built in laptop camera as the default. In both cases, switching to another camera caused the same problem with the same console output.

Sending back to @zepumph , happy to pair on this if that works best!
